src/scaler/spreadscaler/mod.rs

```diff
@@ -211,15 +219,29 @@ impl<S: ReadStore + Send + Sync + Clone> Scaler for ActorSpreadScaler<S> {
                     }
                 } else {
                     // No hosts were eligible, so we can't attempt to add or remove actors
-                    // We will want to return an error to indicate an unsatisfied Scaler here, eventually
-                    trace!("Found no eligible hosts");
+                    trace!(?spread.name, "Found no eligible hosts for spread");
+                    spread_status.push(StatusInfo::failed(&format!("Could not satisfy spread {} for {}, 0/1 eligible hosts found.", spread.name, self.config.actor_reference)));
```

Comment:
0/1? Or just 0? I.e. is only 1 host ever needed?

Reply:
For the actor spreadscaler, a single spread will only ever match with one host since it just picks the first host out of the eligible list (see comment just a bit above)
```
        // NOTE(brooksmtownsend) it's easier to assign one host per list of requirements than
        // balance within those requirements. Users should be specific with their requirements
        // as wadm is not responsible for ambiguity, a future scaler like a DaemonScaler could handle this
```